api/AllocateConference.h

```diff
@@ -10,6 +10,7 @@ struct AllocateConference
 {
     utils::Optional<uint32_t> lastN;
     bool useGlobalPort = true;
+    bool useH264 = false;
```

Comment:
can we do it a little more generic. So if we want to support new video codec ion future (like vp9 or vp10). We will flood this of booleans.

I believe we should have a filed more generic, like:
`videoCodec = "default | vp8 | h264" // default is the one that is on smb config`

If given an unsupported value, http request should fail with 400.

This way we could add support to new codecs without changing API